client/src/components/Ticketing/ticketingmanager/Season/SeasonInstancesPage.tsx

```diff
@@ -2,6 +2,7 @@ import React, {useEffect, useState} from 'react';
 import {useNavigate} from 'react-router-dom';
 import {titleCase} from '../../../../utils/arrays';
 import {getSeasonImage, formatSeasonDate, SeasonImage} from './seasonUtils';
+import GroupToggle from '../../GroupeToggle';
```

Comment:
Just curious, any reason for some being Group and some being Groupe?

Reply:
Thanks for catching the typo. Fixed.